settings/base.py

```diff
@@ -301,6 +302,25 @@ def is_environ_true(name):
             # "SOCKET_CONNECT_TIMEOUT": 5,  # in seconds
             'DB': REDIS_DB,
             'PARSER_CLASS': 'redis.connection.HiredisParser',
+            'REDIS_SECRET_KEY': 'kPEDO_pSrPh3qGJVfGAflLZXKAh4AuHU64tTlP-f_PY=',
+            'CLIENT_CLASS': 'secure_redis.client.SecureDjangoRedisClient',
+            'DATA_RECOVERY': {
+                'OLD_KEY_PREFIX': 'register',
+                'OLD_CACHE_NAME': 'unsafe_redis',
+                'CLEAR_OLD_ENTRIES': False,
+            }
+
+        },
+        'KEY_PREFIX': 'register:secure',
+        'TIMEOUT': 60 * 60 * 24,  # 1 day
+    },
+    'unsafe_redis': {
```

Comment:
@GhalebKhaled Maybe it is better to rename this to something like `plain_values_redis` or `unencrypted_redis`